### PR TITLE
Fix coding standards to fix the build

### PR DESCRIPTION
**Pull request by Zales0123**
| Q               | A
| --------------- | -----
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | 
| License         | MIT

EDIT: I had to upgrade PHPStan to a more modern version (we should've done it a long time ago :dancer:) to make it work with PHP 8.1. Therefore, it resulted in a little bit more changes and skips than expected, but at least it unblocks us with the development 🖖

### CODE REVIEW

File: src/Bundle/Event/ResourceControllerEvent.php [hunk 1]
```diff
@@ -37,7 +37,9 @@ class ResourceControllerEvent extends GenericEvent
 
     private ?Response $response = null;
 
-    /** @psalm-suppress MissingReturnType */
+    /**
+     * @psalm-suppress MissingReturnType
+     */
```
Comment (GSadee) [inline src/Bundle/Event/ResourceControllerEvent.php]: It looked better before change

File: src/Bundle/Grid/Parser/OptionsParser.php [hunk 2]
```diff
@@ -29,13 +29,16 @@ final class OptionsParser implements OptionsParserInterface
...
-        PropertyAccessorInterface $propertyAccessor
+        PropertyAccessorInterface $propertyAccessor,
     ) {
         $this->container = $container;
         $this->expression = $expression;
         $this->propertyAccessor = $propertyAccessor;
     }
 
+    /**
+     * @param null $data
```
Comment (GSadee) [inline src/Bundle/Grid/Parser/OptionsParser.php]:
```suggestion
     * @param mixed $data
```
or even better to list explicitly possible types

File: src/Bundle/Storage/CookieStorage.php [hunk 3]
```diff
@@ -58,6 +58,9 @@ public function onKernelResponse(ResponseEvent $event): void
         }
 
         $response = $event->getResponse();
+        /**
+         * @var string|null $value
+         */
```
Comment (GSadee) [inline src/Bundle/Storage/CookieStorage.php]: One line?

**Review by GSadee — APPROVED** [review]

File: src/Bundle/Event/ResourceControllerEvent.php (hunk 1 above)
Comment (Zales0123) [reply src/Bundle/Event/ResourceControllerEvent.php]: That's... true 😄

**Pull request merged**